src/store/reducers/healthcheck.reducer.js

```diff
@@ -47,7 +47,7 @@ export default function reducer(state = initialState, action) {
         case FETCH_HEALTHCHECK_LIST_SUCCESS: {
             return {
                 ...state,
-                status: checkStatus(action.payload.status),
+                status: checkStatus(action.payload.failures),
```

Comment:
Just use `!action.payload.failures`, no need to write a function for it, because all the function does is applying `!`.

Reply:
F => for functionality 😃